src/codegen/templates/task/_in_stream.py.mako

```diff
@@ -11,12 +11,14 @@ from typing import Type
 import numpy
 import numpy.typing
 import deprecation
+import pathlib
 
 from nidaqmx._task_modules.channels.channel import Channel
 from nidaqmx.utils import unflatten_channel_string
 from nidaqmx.constants import (
     ${', '.join([c for c in enums_used]) | wrap(4, 4)})
 
+from typing import Union
```

Comment:
Add this to line 9. These imports are sorted into 3 sections: standard, 3rd party, nidaqmx.

Reply:
Noted. `Union` is moved up to line 9.